test-runner-backend/terraform/main.tf

```diff
@@ -61,7 +61,7 @@ resource "google_app_engine_application" "firestore" {
 # Create a firewall rule to allow http communication to compute instance.
 resource "google_compute_firewall" "rules" {
   project     = var.project_id
-  name        = "allow-http-firewall-rule"
+  name        = "allow-http-firewall-rule-v2"
```

Comment:
this is a little messy, but this is the current state of the resources in production.

Reply:
This is weird! Queue name conflict is a known problem. But I am not sure why workflow and firewall is not destroyed. I was not facing this, terraform was able to gracefully destroy all the resources except firestore.
If this issue persists then we have to modify github secrets as well for the new names of workflow and queue.